pkgs/by-name/li/libitl/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
```

Comment:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake 
}:
```

Reply:
Is this necessary? Other packages use them in a single line.

Reply:
Single line diffs for subsequent PRs are harder to read. Multiline module inputs will be mandatory when the new formatter is going to be applied Nixpkgswide, albeit with slightly different syntax.

Reply:
@azahi I'll format it like this for now - but I will say that commas before instead of after look really, really terrible

Reply:
A matter of preference. This is notation is used pretty much everywhere in Nixpkgs. You can also use `nixfmt-rfc-style` to automatically format Nix code, this formatter has a slightly different approach for listing function arguments.